internal/cmd/update_test.go

```diff
@@ -0,0 +1,215 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"io/fs"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+
+	"cdr.dev/slog/sloggers/slogtest/assert"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/afero"
+)
+
+const (
+	fakeExePath    = "/coder"
+	fakeCoderURL   = "https://my.cdr.dev"
+	fakeNewVersion = "1.23.4-rc.5+678-gabcdef-12345678"
+	fakeOldVersion = "1.22.4-rc.5+678-gabcdef-12345678"
+	fakeReleaseURL = "https://github.com/cdr/coder-cli/releases/download/v1.23.4/coder-cli-linux-amd64.tar.gz"
+)
+
+func Test_updater_run_noop(t *testing.T) {
+	fakefs := afero.NewMemMapFs()
+	httpClient := &fakeGetter{
```

Comment:
Lots of duped code in these tests, consider making a `run()`-like function like we have in the monorepo